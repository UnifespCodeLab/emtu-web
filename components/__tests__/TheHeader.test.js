// import { mount, createLocalVue } from '@vue/test-utils'
// import Vuetify from 'vuetify'

// import TheHeader from '~/components/TheHeader.vue'

// describe('Components / TheHeader', () => {
//   const localVue = createLocalVue()
//   let vuetify
//   let wrapper

//   beforeEach(() => {
//     vuetify = new Vuetify()

//     wrapper = mount(TheHeader, {
//       localVue,
//       vuetify,
//       mocks: {
//         $route: { }
//       }
//     })
//   })

//   it('matches the screenshot', () => {
//     expect(wrapper.element).toMatchSnapshot()
//   })

//   it('should render title correctly', () => {
//     const title = wrapper.findComponent({ name: 'v-toolbar-title' })
//     expect(title.text()).toBe('VApt')
//   })

//   it('should render bus icon correctly', () => {
//     const logo = wrapper.find('#bus-icon')
//     expect(logo.exists()).toBe(true)
//   })

//   it('should render menu icon in the toolbar', () => {
//     const menu = wrapper.find('#menu-icon')
//     expect(menu.exists()).toBe(true)
//   })

//   it('should toggle SideBar event when menu-icon is clicked', async () => {
//     const menu = wrapper.find('#menu-icon')
//     await menu.trigger('click')
//     expect(wrapper.emitted().toggleSideBar).toBeTruthy()
//   })
// })
