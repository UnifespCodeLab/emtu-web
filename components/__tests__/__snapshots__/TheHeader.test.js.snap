// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components / TheHeader matches the screenshot 1`] = `
<div
  class="v-card v-card--flat v-sheet theme--light rounded-0"
  style="height: 50px;"
>
  <header
    class="v-sheet theme--dark v-toolbar v-toolbar--dense primary"
    style="height: 48px;"
  >
    <div
      class="v-toolbar__content"
      style="height: 48px;"
    >
      <i
        aria-hidden="true"
        class="v-icon notranslate mdi mdi-bus theme--dark"
        id="bus-icon"
      />
       
      <div
        class="v-toolbar__title ml-4"
      >
        
      EMTU
    
      </div>
       
      <div
        class="spacer"
      />
       
      <button
        class="v-app-bar__nav-icon v-btn v-btn--icon v-btn--round theme--dark v-size--default"
        id="menu-icon"
        type="button"
      >
        <span
          class="v-btn__content"
        >
          <i
            aria-hidden="true"
            class="v-icon notranslate mdi mdi-menu theme--dark"
          />
        </span>
      </button>
    </div>
  </header>
</div>
`;
